src/runtime_src/core/tools/common/SubCmd.cpp

```diff
@@ -34,16 +34,18 @@ SubCmd::SubCmd(const std::string & _name,
   , m_isHidden(false)
   , m_isDeprecated(false)
   , m_isPreliminary(false)
+  , m_defaultDeviceValid(true)
 {
   // Empty
 }
 
 void
 SubCmd::printHelp( const boost::program_options::options_description & _optionDescription,
-                   const boost::program_options::options_description & _optionHidden) const
+                   const boost::program_options::options_description & _optionHidden,
+                   bool removeLongOptDashes /*=false*/) const
```

Comment:
Understand why you have the comment here, but it isn't needed in the body of this method.